### PR TITLE
[DOCS-4295] Replace Vector with Observability Pipelines

### PR DESCRIPTION
**Pull request by maycmlee**
### What does this PR do?

- Replace "Vector" with Observability Pipelines
- Add alias, update nav, and links for "Vector Configurations" doc, which is now titled "Configurations"
- Fix gremlins (ie. smart quotes/apostrophes)
- Websites also updated code so that "Aggregating Agents" in the Agent section of the nav, will redirect to the nav link (Guide section) in the Observability Pipelines section.

### Motivation

DOCS-4295, PM request.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/observability_pipelines/guide/configure_observability_pipelines_with_datadog.md [hunk 1]
```diff
@@ -85,19 +83,13 @@ Use the official Datadog Helm chart, and add the [Agent configuration settings](
...
-Vector provides an [official chart for aggregating data][5] that comes with a Datadog
-log source pre-configured. For more information about installing Vector using Helm,
-see the [official Vector documentation][6].
-
-Vector's chart can hold any valid Vector configuration in the `values.yaml` file using the `customConfig` field. To enable `datadog_logs`, these [Vector configurations](#vector-configurations) can be directly included as-is in the Vector chart configuration.
-
-## Vector configurations
+Observability Pipelines Worker's chart can hold any valid configuration in the `values.yaml` file using the `customConfig` field. To enable `datadog_logs`, these [configurations](#observability-pipelines-worker-configurations) can be directly included as-is in the chart configuration.
 
-See the [Vector documentation][7] for all available configuration parameters and transforms that can be applied to all data processed by Vector.
```
Comment (maycmlee) [inline content/en/observability_pipelines/guide/configure_observability_pipelines_with_datadog.md:96]: I will add text and link to the Reference section once that is pulled in.

File: content/en/observability_pipelines/guide/configure_observability_pipelines_with_datadog.md [hunk 2]
```diff
@@ -85,19 +83,13 @@ Use the official Datadog Helm chart, and add the [Agent configuration settings](
 
 For additional details about the Datadog Helm chart, see [the Kubernetes documentation][4].
 
-Vector provides an [official chart for aggregating data][5] that comes with a Datadog
```
Comment (maycmlee) [inline content/en/observability_pipelines/guide/configure_observability_pipelines_with_datadog.md:88]: I will add text and a link to the OPW helm chart once that is available.

**Review by barieom — CHANGES_REQUESTED** [review]

**Review by apigirl — APPROVED** [review]

File: content/en/observability_pipelines/configurations.md [hunk 3]
```diff
@@ -1,32 +1,27 @@
 ---
-title: Vector Configurations
+title: Configurations
 kind: Documentation
 aliases:
   - /integrations/observability_pipelines/vector_configurations/
+  - /observability_pipelines/vector_configurations/
```
Comment (barieom) [inline content/en/observability_pipelines/configurations.md:6]: can we replace this with just /configurations/?

File: content/en/observability_pipelines/guide/configure_observability_pipelines_with_datadog.md [hunk 4]
```diff
@@ -1,80 +1,78 @@
 ---
-title: Integrate Vector with Datadog
+title: Configure Observability Pipelines with Datadog
 kind: documentation
 aliases:
   - /agent/vector_aggregation/
   - /integrations/observability_pipelines/integrate_vector_with_datadog/
+  - /observability_pipelines/integrate_vector_with_datadog/
```
Comment (barieom) [inline content/en/observability_pipelines/guide/configure_observability_pipelines_with_datadog.md:7]: Can we change this to /observability_pipelines_worker_[text]/?

File: content/en/observability_pipelines/guide/configure_observability_pipelines_with_datadog.md [hunk 5]
```diff
@@ -490,26 +479,22 @@ vector:
...
-[14]: https://vector.dev/docs/reference/configuration/sinks/datadog_logs/
-[15]: https://vector.dev/docs/reference/configuration/sinks/datadog_metrics/
-[16]: https://vector.dev/docs/about/concepts/#buffers
-[17]: https://vector.dev/docs/setup/going-to-prod/architecting/#buffering-data
+[5]: https://vector.dev/docs/reference/configuration/sources/datadog_agent/
+[6]: /observability_pipelines/working_with_data
+[7]: /logs/log_configuration/attributes_naming_convention/#reserved-attributes
+[8]: https://vector.dev/docs/reference/vrl/
+[9]: /getting_started/tagging
+[10]: https://vector.dev/docs/reference/configuration/sinks/datadog_logs/
```
Comment (barieom) [inline content/en/observability_pipelines/guide/configure_observability_pipelines_with_datadog.md:497]: Replace with reference docs links

File: content/en/observability_pipelines/guide/control_log_volume_and_size.md [hunk 6]
```diff
@@ -7,22 +7,22 @@ further_reading:
   - link: "/observability_pipelines/working_with_data/"
     tag: "Documentation"
     text: "Working with data using Vector"
```
Comment (barieom) [inline content/en/observability_pipelines/guide/control_log_volume_and_size.md:9]: Replace with "working with data using Observability Pipelines"

File: content/en/observability_pipelines/guide/control_log_volume_and_size.md [hunk 7]
```diff
@@ -493,8 +493,8 @@ del(.unecessary_tag_field)"""
 
 {{< partial name="whats-next/whats-next.html" >}}
 
-[1]: /observability_pipelines/setup/
-[2]: /observability_pipelines/vector_configurations/
+[1]: /observability_pipelines/installation/
+[2]: /observability_pipelines/configurations/
 [3]: https://vector.dev/docs/reference/configuration/transforms/dedupe/
```
Comment (barieom) [inline content/en/observability_pipelines/guide/control_log_volume_and_size.md:498]: replace with reference docs

File: content/en/observability_pipelines/working_with_data.md [hunk 8]
```diff
@@ -25,23 +25,23 @@ further_reading:
...
 
-Vector enables you to shape and transform observability data. Similar to Logging without Limits™ pipelines, you can configure Vector pipelines that are composed of a series of Vector `transform` components. These transforms allow you to parse, structure, and enrich data with built-in type safety.
+Observability Pipelines enables you to shape and transform observability data. Similar to Logging without Limits™ pipelines, you can configure pipelines for Observability Pipelines that are composed of a series of `transform` components. These transforms allow you to parse, structure, and enrich data with built-in type safety.
 
 ## Remap data with Vector Remap Language
 
 Vector Remap Language (VRL) is an expression-oriented, domain specific language designed for transforming observability data (logs and metrics). It features a simple syntax and [built-in functions][1] tailored to observability use cases.
 
-Vector Remap Language is supported in Vector’s `remap` transform. 
+Vector Remap Language is supported in the `remap` transform.
```
Comment (barieom) [inline content/en/observability_pipelines/working_with_data.md:44]: Let's replace this with "VRL"

File: content/en/observability_pipelines/working_with_data.md [hunk 9]
```diff
@@ -402,7 +402,7 @@ The `threshold` field defines the number of events allowed for a given bucket. `
 [12]: https://vector.dev/docs/reference/vrl/functions/#event-functions
```
Comment (barieom) [inline content/en/observability_pipelines/working_with_data.md:402]: Replace all links with reference docs

File: content/en/observability_pipelines/working_with_data.md [hunk 10]
```diff
@@ -25,23 +25,23 @@ further_reading:
...
-    text: Learn more about Vector configurations
+    text: Learn more about Observability Pipelines configurations
 ---
 
 ## Overview
 
-Vector enables you to shape and transform observability data. Similar to Logging without Limits™ pipelines, you can configure Vector pipelines that are composed of a series of Vector `transform` components. These transforms allow you to parse, structure, and enrich data with built-in type safety.
+Observability Pipelines enables you to shape and transform observability data. Similar to Logging without Limits™ pipelines, you can configure pipelines for Observability Pipelines that are composed of a series of `transform` components. These transforms allow you to parse, structure, and enrich data with built-in type safety.
 
 ## Remap data with Vector Remap Language
```
Comment (barieom) [inline content/en/observability_pipelines/working_with_data.md:40]: could we change this to "VRL" so "Vector" is less prominent in the title?

**Review by barieom — CHANGES_REQUESTED** [review]

File: content/en/observability_pipelines/_index.md [hunk 11]
```diff
@@ -4,65 +4,60 @@ kind: Documentation
...
     text: Set up Observability Pipelines 
   - link: https://www.datadoghq.com/blog/datadog-observability-pipelines/
     tag: Blog
     text: Take control of your telemetry data with Observability Pipelines
-  - link: /observability_pipelines/vector_configurations/
+  - link: /observability_pipelines/configurations/
     tag: Documentation
-    text: Learn more about Vector configurations
+    text: Learn more about Observability Pipelines configurations
   - link: https://vector.dev/docs/setup/going-to-prod/
```
[the remark this answers is not in the document]
Comment (maycmlee) [reply content/en/observability_pipelines/_index.md:16]: The Going to prod docs haven't been merged yet, I can update this once its merged.

File: content/en/observability_pipelines/configurations.md (hunk 3 above)
Comment (maycmlee) [reply content/en/observability_pipelines/configurations.md:6]: This is an alias, meaning that it's just redirecting the page that used to live at `/observability_pipelines/vector_configurations/` to /observability_pipelines/configurations/

File: content/en/observability_pipelines/guide/configure_observability_pipelines_with_datadog.md (hunk 4 above)
Comment (maycmlee) [reply content/en/observability_pipelines/guide/configure_observability_pipelines_with_datadog.md:7]: This is an alias.

**Pull request merged**